src/lightning/fabric/utilities/seed.py

```diff
@@ -7,13 +7,18 @@
 
 import numpy as np
 import torch
+from lightning_utilities.core.imports import RequirementCache
 
 from lightning.fabric.utilities.rank_zero import _get_rank, rank_prefixed_message, rank_zero_only, rank_zero_warn
 
 log = logging.getLogger(__name__)
+_NUMPY_AVAILABLE = RequirementCache("numpy")
```

Comment:
We keep an inventory of these variables in fabric/utilities/imports.py, let's move it there and import it here :)

Reply:
Suggestion mentioned above implemented, Please check.